Audiobook Player/PlayerManager.swift

```diff
@@ -10,183 +10,180 @@ import Foundation
 import AVFoundation
 import MediaPlayer
 
+// swiftlint:disable function_body_length file_length
+
 class PlayerManager: NSObject {
     static let sharedInstance = PlayerManager()
-    let defaults:UserDefaults = UserDefaults.standard
-    
+    let defaults: UserDefaults = UserDefaults.standard
+
     //current item to play
-    var playerItem:AVPlayerItem!
-    var currentBook:Book!
-    
-    var fileURL:URL!
-    
-    var audioPlayer:AVAudioPlayer?
-    
+    var playerItem: AVPlayerItem!
+    var currentBook: Book!
+
+    var fileURL: URL!
+
+    var audioPlayer: AVAudioPlayer?
+
     //chapters
-    var chapterArray:[Chapter] = []
-    var currentChapter:Chapter?
-    
+    var chapterArray: [Chapter] = []
+    var currentChapter: Chapter?
+
     //book identifier for `NSUserDefaults`
-    var identifier:String!
-    
+    var identifier: String!
+
     //speed
-    var currentSpeed:Float = 1.0
-    
+    var currentSpeed: Float = 1.0
+
     //timer to update labels about time
-    var timer:Timer!
-    
+    var timer: Timer!
+
     func isLoaded() -> Bool {
-        guard let _ = self.audioPlayer else {
-            return false
+        if self.audioPlayer != nil {
```

Comment:
let's do `return self.audioPlayer != nil`

Reply:
Oh I wasn't sure that was an option and I didn't try 😄 will update.